api.planx.uk/modules/pay/helpers.ts

```diff
@@ -1,6 +1,9 @@
+import { getFeeBreakdown } from "@opensystemslab/planx-core";
+import type { FeeBreakdown } from "@opensystemslab/planx-core/types";
 import { gql } from "graphql-request";
 import airbrake from "../../airbrake.js";
 import { $api } from "../../client/index.js";
+import { getSessionData } from "../send/email/service.js";
```

Comment:
We don't really have a good pattern in the API for sharing these common queries (there's a few bundles up in `helpers.ts` but I think that's it).

It's probably worth moving this to a shared location if it's being used across modules - somebody working in `/send` could update this function without considering the impact of this on other sections of the codebase.

Reply:
Fair point - instead of moving to a shared spot, actually just defined a fresh `getPassportData` query direct in this file (it's a good excuse to use graphql's `path` !)